config/webpack.dev.config.js

```diff
@@ -72,12 +73,19 @@ module.exports = Merge.smart(commonConfig, {
       inject: true, // Appends script tags linking to the webpack bundles at the end of the body
       template: path.resolve(__dirname, '../public/index.html'),
     }),
+    // when the --hot option is not passed in as part of the command
+    // the HotModuleReplacementPlugin has to be specified in the Webpack configuration
+    // https://webpack.js.org/configuration/dev-server/#devserver-hot
+    new webpack.HotModuleReplacementPlugin(),
   ],
   // This configures webpack-dev-server which serves bundles from memory and provides live
   // reloading.
   devServer: {
     host: '0.0.0.0',
     port: 1991,
     historyApiFallback: true,
+    hot: true,
+    inline: true,
+    progress: true,
```

Comment:
@jaebradley Looking at the [Webpack documentation](https://webpack.js.org/configuration/dev-server/#devserver-progress-cli-only), it looks like the `progress` config setting is CLI only.